tests/nightly/nvidia-drivers-compat/task.yaml

```diff
@@ -0,0 +1,39 @@
+summary: Collect and check nvidia driver libraries
+
+details: |
+  The test collects information about userspace libraries shipped by various
+  nvidia driver releases across a number of Ubuntu releases.
+
+# it's enough to run on a single host as the test launches all other releases in
+# containers
+
+systems: [ubuntu-24.04-64]
+
+kill-timeout: 60m
+
+execute: |
+  snap install lxd
+  lxd init --auto
+  lxd waitready
```

Comment:
I think we have a bit of logic to handle cleanup post the use of LXD.
Can you include this in prepare somewhere.

``` sh
"$TESTSTOOLS"/lxd-state prepare-snap
tests.cleanup defer "$TESTSTOOLS"/lxd-state undo-mount-changes
```

Additionally the snap install and lxd init should probably move to prepare?